[2024-06-05T00:33:27.616+0800] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-05T00:33:27.632+0800] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: centaline_scraping_dag.centaline_scraping_rent_data manual__2024-06-04T16:33:25.713585+00:00 [queued]>
[2024-06-05T00:33:27.650+0800] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: centaline_scraping_dag.centaline_scraping_rent_data manual__2024-06-04T16:33:25.713585+00:00 [queued]>
[2024-06-05T00:33:27.651+0800] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2024-06-05T00:33:27.675+0800] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): centaline_scraping_rent_data> on 2024-06-04 16:33:25.713585+00:00
[2024-06-05T00:33:27.677+0800] {standard_task_runner.py:63} INFO - Started process 4099 to run task
[2024-06-05T00:33:27.678+0800] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'centaline_scraping_dag', 'centaline_scraping_rent_data', 'manual__2024-06-04T16:33:25.713585+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/cenraline_fc.py', '--cfg-path', '/tmp/tmp5ag5j2_g']
[2024-06-05T00:33:27.679+0800] {standard_task_runner.py:91} INFO - Job 6: Subtask centaline_scraping_rent_data
[2024-06-05T00:33:27.730+0800] {task_command.py:426} INFO - Running <TaskInstance: centaline_scraping_dag.centaline_scraping_rent_data manual__2024-06-04T16:33:25.713585+00:00 [running]> on host LAPTOP-3QN476HM.
[2024-06-05T00:33:27.853+0800] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Ryanng' AIRFLOW_CTX_DAG_ID='centaline_scraping_dag' AIRFLOW_CTX_TASK_ID='centaline_scraping_rent_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-04T16:33:25.713585+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-04T16:33:25.713585+00:00'
[2024-06-05T00:33:27.853+0800] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-05T00:33:28.672+0800] {logging_mixin.py:188} INFO - Fetching offset: 0
[2024-06-05T00:33:29.749+0800] {logging_mixin.py:188} INFO - Fetching offset: 24
[2024-06-05T00:33:29.927+0800] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-05T00:33:29.927+0800] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ryanng/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/ryanng/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/ryanng/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/ryanng/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/ryanng/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/mnt/c/Users/user/Desktop/new_airflow/dags/centaline_scraping.py", line 101, in centaline_scraping_rent_data
    engine = create_engine('postgresql://postgres:1234@localhost:5432/centaline_pj')
  File "<string>", line 2, in create_engine
  File "/home/ryanng/.local/lib/python3.10/site-packages/sqlalchemy/util/deprecations.py", line 375, in warned
    return fn(*args, **kwargs)
  File "/home/ryanng/.local/lib/python3.10/site-packages/sqlalchemy/engine/create.py", line 544, in create_engine
    dbapi = dialect_cls.dbapi(**dbapi_args)
  File "/home/ryanng/.local/lib/python3.10/site-packages/sqlalchemy/dialects/postgresql/psycopg2.py", line 811, in dbapi
    import psycopg2
ModuleNotFoundError: No module named 'psycopg2'
[2024-06-05T00:33:29.939+0800] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=centaline_scraping_dag, task_id=centaline_scraping_rent_data, run_id=manual__2024-06-04T16:33:25.713585+00:00, execution_date=20240604T163325, start_date=20240604T163327, end_date=20240604T163329
[2024-06-05T00:33:29.955+0800] {standard_task_runner.py:110} ERROR - Failed to execute job 6 for task centaline_scraping_rent_data (No module named 'psycopg2'; 4099)
[2024-06-05T00:33:29.978+0800] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-05T00:33:29.992+0800] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-05T00:33:29.992+0800] {local_task_job_runner.py:222} INFO - ::endgroup::
